internal/action/bufpane.go

```diff
@@ -351,16 +349,15 @@ func (h *BufPane) resetMouse() {
 func (h *BufPane) OpenBuffer(b *buffer.Buffer) {
 	h.Buf.Close()
 	h.Buf = b
+	// Set the OverwriteModeActive by default to false on open
+	h.Buf.OverwriteModeActive = false
```

Comment:
BTW this assignment itself is not too useful either, since it is initialized to false by default anyway.